lib/src/value/color.ts

```diff
@@ -154,6 +155,8 @@ function encodeSpaceForColorJs(space?: KnownColorSpace): string | undefined {
       return 'a98rgb';
     case 'display-p3':
       return 'p3';
+    case 'display-p3-linear':
```

Comment:
Nit: Do we have [exhaustive checks](https://typescript-eslint.io/rules/switch-exhaustiveness-check/) enabled in eslint?

It doesn't seem like we do so explicitly, it's not enabled by default by eslint, and I couldn't tell whether `gts` enabled it 🤔 

If not, it may be a good idea to do so, or at the very least to have a `const _never: never = value` in a default case for the compiler to warn about it.

Reply:
Looks like we do have them enabled.